config/routes.rb

```diff
@@ -25,4 +24,10 @@
 
   get "/auth/github", as: :github_login
   get "/auth/github/callback", to: "external_sessions#create"
+
+  namespace :api do
+    namespace :v1 do
+      get "/price-comparison/:query", to: "price_comparison#index"
```

Comment:
Do we want price-comparison, or price_comparison? Either way let's be consistent